openslides/assignments/static/js/assignments/site.js

```diff
@@ -470,6 +471,12 @@ angular.module('OpenSlidesApp.assignments.site', [
                 $scope.defaultProjectorId = projectiondefault.projector_id;
             }
         });
+        $scope.$watch(function () {
```

Comment:
If we use such a watcher here we do not need the Assignment.bindOne in line 464. You have to extend the watch callback. See http://www.js-data.io/docs/js-data-angular#dsbindone